src/atoms/dialog.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { Button, Dialog as MuiDialog, DialogActions, DialogContent, DialogContentText, DialogProps as MuiDialogProps, DialogTitle } from '@mui/material';
+
+interface DialogProps extends Omit<MuiDialogProps, 'content' | 'title'> {
+  actions?: React.ReactNode;
+  buttonType?: 'button' | 'submit'; 
+  content: React.ReactNode | string | undefined;
+  label: string;
+  size: 'sm' | 'md' | 'lg';
+  title: React.ReactNode | string;
+  type: 'alert' | 'form' | 'size';
```

Comment:
size type might not make much sense as a separate dialog type if its only purpose is to adjust the size of the dialog. Instead of treating size as a dialog type, you could leave it alone as a property that applies to all dialog types.
What do you think?